trunk/SUAVE/Components/Energy/Converters/Propeller.py

```diff
@@ -200,26 +204,25 @@ def spin(self,conditions):
         
         #Things that will change with iteration
         size = (len(a),N)
-    
+        Cl = np.zeros((1,N))  
+        
         #Setup a Newton iteration
         psi    = np.ones(size)
         psiold = np.zeros(size)
         diff   = 1.
         
         ii = 0
+        broke = False   
         while (diff>tol):
             sin_psi = np.sin(psi)
             cos_psi = np.cos(psi)
             Wa      = 0.5*Ua + 0.5*U*sin_psi
             Wt      = 0.5*Ut + 0.5*U*cos_psi   
-            #va     = Wa - Ua
+            va     = Wa - Ua
```

Comment:
Not here, but it will be used in the upcoming PR where I change the uniform inflow model with BET. As @timdmacdo suggested earlier, we want to avoid having lines of code commented out because it gives a misleading impression